Sasha/Files/csv_sort/test_csv_sort.py

```diff
@@ -0,0 +1,61 @@
+from csv_sort import sort_data
+from csv_sort import open_file
+import unittest
+import csv
+
+class TestCSVSort(unittest.TestCase):
```

Comment:
I don't see tests that are checking valid sorting.
You can create additional files for tests in another folder like 'test-resources'